mythril/laser/ethereum/state/constraints.py

```diff
@@ -30,6 +30,12 @@ def __init__(
         self._default_timeout = 100
         self._is_possible = is_possible
 
+    def compress(self):
```

Comment:
```suggestion
    def as_smt_bool(self):
```
I think this is more descriptive of what the function does